.travis.yml

```diff
@@ -1,53 +1,58 @@
 language: php
 sudo: false
+dist: trusty
```

Comment:
According to this post (https://blog.travis-ci.com/2017-07-11-trusty-as-default-linux-is-coming?utm_source=web&utm_medium=banner&&utm_campaign=trusty-default) the migration shall be finished at 1st of september.